tests/opentelemetry-test-utils/src/opentelemetry/test/test_base.py

```diff
@@ -85,6 +90,61 @@ def sorted_spans(self, spans):  # pylint: disable=R0201
             reverse=True,
         )
 
+    def sorted_metrics(self, metrics):  # pylint: disable=R0201
+        """
+        Sorts metrics by metric name.
+        """
+        return sorted(
+            metrics,
+            key=lambda m: m.name,
+        )
+
+    def get_sorted_metrics(self):
+        resource_metrics = (
+            self.memory_metrics_reader.get_metrics_data().resource_metrics
+        )
+        for metrics in resource_metrics:
+            for scope_metrics in metrics.scope_metrics:
+                all_metrics = list(scope_metrics.metrics)
+                return self.sorted_metrics(all_metrics)
```

Comment:
This returns to the first resource and scope; what happens to the rest?

Reply:
Fixed